docs/Validation.md

```diff
@@ -417,3 +417,66 @@ If `xml.validation.resolveExternalEntities` is set to `true` (default is `false`
 Demonstration of the different behaviour:
 
 ![When using the default settings, an external entity that has an XML element as its content will not produce a validation error when nested in an element that expects character content. If `xml.validation.resolveExternalEntities` is enabled, then an error will be produced](./images/Validation/ExternalEntityResolvingDemonstration.gif)
+
+## xml.validation.schema.enabled
+
+The `xml.validation.schema.enabled` gives the capability to enable / disable the validation based on XSD. It can be configured with 3 values:
+
+ * `always`: enable schema based validation.
+ * `never`: disable schema based validation.
+ * `onValidSchema`: enable schema based validation only when the declared xsi:schemaLocation hint or xsi:noNamespaceSchemaLocation is valid for the root element.
+
+To understand the `onValidSchema` settings value, takes a sample:
```

Comment:
I think it would make more sense if you changed the wording of "takes a sample" to "lets go through an example"